tests/test_bagit_archive.py

```diff
@@ -0,0 +1,182 @@
+import shutil
+import zipfile
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+import bagit
+import pytest
+
+from apt.bagit_archive import BagitArchive
+
+
+class TestBagitArchive:
+    @pytest.fixture
+    def sample_file_path(self, tmp_path):
+        """Return path to sample file fixture."""
+        source_path = Path(__file__).parent / "fixtures" / "sample.txt"
+        target_path = tmp_path / "sample.txt"
+        target_path.write_text(source_path.read_text())
+        return target_path
+
+    @pytest.fixture
+    def input_files(self, sample_file_path):
+        """Create a list of input files for testing."""
+        return [
+            {
+                "uri": str(sample_file_path),
+                "filepath": "documents/sample.txt",
+                "checksums": {
+                    "md5": "a5e8cf45ba8f5f6df3823df3d0e452fa",
+                    "sha256": "e71709bc3a8a2a9c602bcd8a35eefd56b3a4a38a10a06a75",
+                },
+            }
+        ]
+
+    def test_init_defaults(self):
+        ba = BagitArchive()
```

Comment:
Is it worth making this a fixture given how times it is repeated? And maybe a more descriptive variable name?

Reply:
My argument for keeping as-is would be a) that it's instantiated differently from time-to-time and b) simplicity.  If we had a fixture for just `BagitArchive()` without args, it'd feel weird when we _do_ have args and then don't have a fixture.

Maybe it'll make sense if the project / tests grow considerably, but feels okay at this scale to me.

While I think _consistent_ short variable names can be okay (e.g. `df` for a generic pandas dataframe in a test), particularly in tests, renamed to `bagit_archive` in tests.  That way, it'll match in code where we would probaly prefer `bagit_archive` to `ba`.

Reply:
Sounds good!